m4_sessions/1806-inning/group7.md

```diff
@@ -1,33 +1,38 @@
 ## Group Members:
+Jim Dechant, Tony Robertson, Tyler Lundgren, Tyler McNamara
 
 
-
-## Name of Session: 
+## Name of Session: Lets just fuckin relax!
 
 ## Session Description 
 
-(Why should students come to this session): Are you interested in biological enhancement? Drop in to learn about biohacking.
+ It's week 5, you are not stressed at all, but if you are this is the gear up for you. ( We will be stretching and meditating but isn't required )
 
 ## Session Outcomes 
 
-(What will students learn/leave with?): 
+This is mainly to take some time to yourself and relax, and hopefully you will have a few tools to take with you and use in times of stress. 
 
 ## Session Outline 
 
-(Please include who is facilitating and how much time you are going to spend in each section):
+Tyler L. => Everyone => Our Introductions - 5 min
```

Comment:
Would love for you to add detail to each of these:
What is your intro?
What can you do on the way to the park?
Can you list the stretches?  
Can you write out the guided meditation?